src/main/java/org/fundacionjala/coding/nestor/SortInnerContent.java

```diff
@@ -0,0 +1,42 @@
+package org.fundacionjala.coding.nestor;
+
+import java.util.Arrays;
+import java.util.stream.Collectors;
+
+/**
+ * Class to sort inner content.
+ */
+public class SortInnerContent {
+
+  /**
+   * Main method.
+   *
+   * @param words type string.
+   * @return a new string with inner ordered.
+   */
+  public String sortTheInnerContent(final String words) {
+    if (words.length() > 0) {
+      return Arrays.stream(words.split(" "))
+        .map(e -> e.length() > 1 ? new StringBuilder()
+          .append(e.substring(0, 1))
+          .append(new StringBuilder(sortString(e.substring(1, e.length() - 1))).reverse().toString())
+          .append(e.substring(e.length() - 1)) : e
+        )
+        .collect(Collectors.joining(" "));
+    }
+
+    return words;
```

Comment:
This may be refactored like this.
```java
  public String sortTheInnerContent(final String words) {
      return Arrays.stream(words.split(" "))
        .map(e -> e.length() > 1 ? new StringBuilder()
          .append(e, 0, 1)
          .append(sortString(e.substring(1, e.length() - 1)))
          .append(e.substring(e.length() - 1)) : e)
        .collect(Collectors.joining(" "));
  }

  /**
   * Helper to sort inner container string.
   *
   * @param inputString Type String.
   * @return string ordered.
   */
  private String sortString(final String inputString) {
    String[] tempArray = inputString.split("");
    Arrays.sort(tempArray, Comparator.reverseOrder());
    return String.join("", tempArray);
  }
```